zqd/search/search.go

```diff
@@ -28,7 +28,9 @@ const StatsInterval = time.Millisecond * 500
 
 const (
 	MimeTypeCSV    = "text/csv"
+	MimeTypeJSON   = "application/x-ndjson"
```

Comment:
Should this be be `application/json` ?

Reply:
That's what was in my head but not in my typing fingers.  :-)